portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -2016,4 +2016,55 @@ definition {
 			title = "WC Title");
 	}
 
+    
+	@description = "LPS-117341 Create a label for one of the categories and add another one with the text "..." and tooltip or popover with all the categories left"
+	@priority = "5"
+	test CheckForMoreCategories {
+	    property portal.acceptance = "true";
+
+     	Category.openCategoriesAdmin(siteURLKey = "global");
+
+        task ("Create a list of Audiences"){
+			for (var count : list "1,2,3"){
+				ContentDashboard.addAudience(
+					title  = "teste ${count}");
+			}		
+		}
```

Comment:
Why do you create a new method when you can use JSON macro to create categories?